test/output/fixable.json

```diff
@@ -1,5 +1,5 @@
 {
-    "basePath": "/v1",
+    "basePath": "/v1/",
```

Comment:
Well, it's an `iojs` specific issue. I'm guessing it has to do something with `url` module and breaking changes in `iojs`